libraries/AP_NavEKF3/AP_NavEKF3.cpp

```diff
@@ -719,6 +719,14 @@ const AP_Param::GroupInfo NavEKF3::var_info2[] = {
     // @Increment: 1
     // @User: Advanced
     AP_GROUPINFO("PRIMARY", 8, NavEKF3, _primary_core, EK3_PRIMARY_DEFAULT),
+
+    // @Param: LOG_LEVEL
```

Comment:
In AIS I called it `AIS_LOGGING`. I think this is better. But we should pick a good name and use that throughout if were going to make a new pattern. Happy to change AIS to match this.